### ISSUE TITLE
[Icon] Remove the `"img"` `role` (CAL-AMP15)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13528]
**monday.com sync:** #10822034899

### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) 

### Severity: 
 High 

### Recommended solution
- `calcite-icon` should not have an associated role, and the `role="img"` should be removed. 
- Estimate includes a test to confirm no role is on the component.
- All components inheriting `calcite-icon` should not have the `role="img"`

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP15 Ensure ARIA roles, states, and properties are valid 
- Module: M2: Filter results with regions_Earthquake results 
- Finding ID: [CAL-AMP15](https://esri.hub.essentia11y.com/short-link/sKVMEfolM10QZ27H) 

**Issue**
There are elements with an unnecessary role="img" attribute. Examples include:
- Magnitude 5.6
- Magnitude 0.9
- Magnitude 5.5
- Magnitude 1.5 and so on.

**User Impact**
Assistive technologies may be unable to parse this content, so users may not be able to perceive the feature(s) or control(s) correctly, or may not be able to operate them.

**Code Reference** 
 ```
<div class=" container text--slotted non-interactive " role="img" tabindex="-1"><!--?lit$219689914$-->(...)</div>

```

**Steps to Reproduce**
1. Open Chrome DevTools.
2. Locate the element identified in the code reference.
3. Notice that the element's ARIA attributes and/or nesting do not conform to the ARIA specification: https://www.w3.org/TR/wai-aria/ and/or the ARIA in HTML specification: https://www.w3.org/TR/html-aria/
 
**Recommendation**
Ensure ARIA roles, states, and properties are valid. 

In this case, the developer should remove role="img" from the ```<div>``` element.

**Compliant Code Example**
```
<img src="oldbarn.png" alt="Old barn near Jenny Lake" aria-details="imgDetails">
...
<p id="imgDetails">
An old barn with a tin roof and weathered siding stands bathed in morning sunlight near Jenny Lake with the Wyoming Rocky Mountains in the background.
</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP15.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/sKVMEfolM10QZ27H) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by geospatialem** [comment]
Chatted with @jcfranco on this, and since we are applying a label to the component the role is needed in this case, and may have been flagged as a result of shadow DOM. Closing the above out as unplanned.
